### PR TITLE
Slovakia Flags

### PR DESCRIPTION
**Pull request by miki-g**

### CODE REVIEW

File: NetKAN/KSP-Slovakia-flags [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "spec_version" : "v1.2",
+    "identifier"   : "KSP Slovakia flags",
+    "$kref"        : "#/ckan/github/miki-g/KSP-Slovakia-Flags",
+    "$vref"        : "#/ckan/ksp-avc",
```
Comment (politas) [inline NetKAN/KSP-Slovakia-flags]: Can you replace this line with the following?

    "ksp_version"    : "any",
    "comment"        : "flagmod",

File: NetKAN/KSP-Slovakia-flags [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "spec_version" : "v1.2",
+    "identifier"   : "KSP Slovakia flags",
```
Comment (politas) [inline NetKAN/KSP-Slovakia-flags]: identifier should be "KSP-Slovakia-flags", matching the filename

File: NetKAN/KSP-Slovakia-flags (hunk 1 above)
Comment (miki-g) [inline NetKAN/KSP-Slovakia-flags]: "Can you replace this line with the following?" - This means which line?

in coment is 4 lines, from 2 to 5

(sry, github and English do not know much)

Comment (Olympic1) [inline NetKAN/KSP-Slovakia-flags]: Replace line 5 with the text in the grey rectangle

**Comment by Olympic1** [comment]
Give filename the extention `.netkan`

**Comment by politas** [comment]
FATAL CKAN.NetKAN.Program (null) - Module contains no files to install.

Hmmmm

File: NetKAN/KSP-Slovakia-flags.netkan [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "ksp_version"    : "any",
+    "comment"        : "flagmod",
+    "name"         : "KSP Slovakia flags",
+    "author"       : [ "Miki" ],
+    "abstract"     : "Slovakia Flags",
+    "license"      : "WTFPL",
+
+    "install": [
+          {
+           "file" : "KSP Slovakia flags",
```
Comment (politas) [inline NetKAN/KSP-Slovakia-flags.netkan]: "file" : "KSP Slovakia Flags",

should work. Case matters for Linux file systems.

Comment (miki-g) [inline NetKAN/KSP-Slovakia-flags.netkan]: hm,
problem with spaces i name? or?

Comment (politas) [inline NetKAN/KSP-Slovakia-flags.netkan]: Capital "F" in Flags in the directory name in the zip file.

Comment (miki-g) [inline NetKAN/KSP-Slovakia-flags.netkan]: O.o
/o\
o.O
:/

thanks

**Comment by politas** [comment]
Merged with thanks!

**Pull request merged**